packages/a11y/src/Skiplink.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import { cleanup, render } from "@testing-library/react"
+import * as React from "react"
+import { SkipLink } from "./SkipLink"
+
+afterEach(cleanup)
+
+describe("<SkipLink />", () => {
+  it("renders a SkipLink link as an anchor tag", () => {
```

Comment:
seems like a bit of a strange test, in no situations would this ever _not_ be an anchor?

Reply:
Guessing the answer is no, but Is there a way that we could test the actually functionality?

Reply:
Hmm... I can test that it receives focus, but I feel ick about testing if it's visuallyHidden with styles.